### PR TITLE
1.1.0 changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Updated screenshots
- Added release notes
- Hid the widget version from the SS

### CODE REVIEW

File: docs/res/setup-guide-done.png
Comment (cs-suyog-jinturkar) [inline docs/res/setup-guide-done.png:1]: wrongly updated?

File: docs/res/setup-guide-expanded.png
Comment (cs-suyog-jinturkar) [inline docs/res/setup-guide-expanded.png:1]: same as abovE?

File: docs/res/setup-guide-launch-point.png
Comment (cs-suyog-jinturkar) [inline docs/res/setup-guide-launch-point.png:1]: same as above

File: docs/res/setup-guide-na.png
Comment (cs-suyog-jinturkar) [inline docs/res/setup-guide-na.png:1]: same as above

File: docs/res/setup-guide-done.png
Comment (cs-abhishek-shukla) [reply docs/res/setup-guide-done.png:1]: Apart from image size, everything else seems to be in order. The version number has been grayed out, though.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Comment by cs-suyog-jinturkar** [comment]
LGTM

File: release_notes.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## What's New
+
+### Enhanced Task Flow in Streamline Section
+
+The **Streamline** section now offers a smoother setup experience with optimized task ordering. Start by configuring indicator extraction &mdash; it's now the first step to help you get off to a powerful start.
+
+- **New First Task**: *Configure Indicator Extraction*  
+   Dive into your setup by prioritizing indicator extraction, now in its most prominent position.
+
+   - **Updated Launch Button**: *Configure Exclude List* is now **Configure Indicator Extraction and Exclusion**
```
Comment (cs-rohit-khune) [inline release_notes.md]: @cs-abhishek-shukla , Please replace "Configure Indicator Extraction and Exclusion" with "Manage Indicator Exclusion List"

image

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

**Review by cs-rohit-khune — APPROVED** [review]

**Pull request merged**